README.md

```diff
@@ -1,46 +1,162 @@
-# Getting Started with Create React App and Redux
+# Project Name :  Crypto-Hub App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+>[Crypto-Hub](crypto-hub-247.netlify.app) is a web application for a cryptocurrencies that provides information about the top 100 most used cryptocurrencies in the worlds.
 
-## Available Scripts
+## Screen Shots
+#### Home Page
+![Home page](cryptoPage1.png)
 
-In the project directory, you can run:
+#### Detail Page
+![Home page](cryptoPage2.png)
 
-### `npm start`
+## Video Description of the project
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[watch here](https://www.loom.com/share/e872a6b13cf24e2ca9eb1718f29cee24)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+## Built With
+> In order to achieve this project I used some resources and technologies listed below :
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- Major languages : JavaScript , JSX and CSS
+- Toolchain : `Babel`
+- Library and Framework : `React, React-redux, Jest, React-test-renderer`
+- Technologies used : Github, Git, Gitflow and Linters
 
-### `npm run build`
+## Live Demo
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+ - Click [Here](crypto-hub-247.netlify.app) To open the live demo
```

Comment:
- [ ] Kindly be noticed that when you wrote the link this way it doesn't work you should write it like this https://crypto-hub-247.netlify.app/.